MODULE.bazel

```diff
@@ -0,0 +1,29 @@
+module(
+    name = "remoteapis",
```

Comment:
i pulled the non-dash version because that's what bazel uses, any precedent for folks using the other one? I didn't need to mirror the repo_name since it's never accessed